repos/system_upgrade/el8toel9/actors/checkdeprecatedrpmsignature/libraries/checkdeprecatedrpmsignature.py

```diff
@@ -0,0 +1,77 @@
+from leapp import reporting
+from leapp.libraries.stdlib import api
+from leapp.models import CryptoPolicyInfo, InstalledRPM, TargetUserSpacePreupgradeTasks
+
+FMT_LIST_SEPARATOR = '\n    - '
+SUMMARY_FMT = (
+    'The SHA-1 hash algorithm is considered unsafe and it is no longer'
+    ' allowed to be used on RHEL 9 systems using the default crypto policy.'
+    ' This causes issues when using DNF/RPM to handle packages with RSA/SHA1'
+    ' signatures as the signature cannot be checked with the default crypto'
+    ' policy. Any such packages cannot be installed, removed,'
```

Comment:
```suggestion
    ' policy. Any such package cannot be installed, removed,'
```

Reply:
`such a package` or `any such packages`. In this case I would like to stick with plural.